### PR TITLE
RT-2.1 and RT-2.2 Updating configuration based on ExplicitInterfaceInDefaultVRF deviation

### PR DESCRIPTION
**Pull request by trathod1**
Following changes are clubbed together as they are interdependent -
  1. As interface must be part of VRF, before it is attached to ISIS -
              - Moving `deviations.ExplicitInterfaceInDefaultVRF` config block from `PushAndStart` to `PushDUT`. 
              - Passing testing.TB handle from PushAndStart to PushDUT 
              - Updating PushDUT calls to pass `testing.TB` handle
  2. Adding `deviation.ExplicitPortSpeed` config as part of this under `PushDUT` 
              - Considering `testing.TB` handle must be passed to `SetPortSpeed`, this deviation function is also updated. (Discussed in https://github.com/openconfig/featureprofiles/pull/948#issuecomment-1385848941)
  3. Updating interface-name usage in all the places based on `deviations.ExplicitInterfaceInDefaultVRF`
  5. Explicitly configuring `Global.LevelCapability` to Level_2, considering different vendor have different default.

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

File: feature/experimental/isis/ate_tests/internal/session/session.go [hunk 1]
```diff
@@ -230,27 +235,32 @@ func (s *TestSession) ConfigISIS(ocFn func(*oc.NetworkInstance_Protocol_Isis), a
...
-		fptest.AssignToNetworkInstance(t, s.DUT, s.DUTPort2.Name(), *deviations.DefaultNetworkInstance, 0)
-	}
 	s.PushAndStartATE(t)
 	return nil
 }
 
 // PushDUT replaces DUT config with s.dutConf. Only interfaces and the ISIS
 // protocol are written.
-func (s *TestSession) PushDUT(ctx context.Context) error {
+func (s *TestSession) PushDUT(t testing.TB, ctx context.Context) error {
```
Comment (xw-g) [inline feature/experimental/isis/ate_tests/internal/session/session.go:247]: plz fix the go format issue.

Comment (trathod1) [reply feature/experimental/isis/ate_tests/internal/session/session.go]: Committed changes to fix go format.

**Comment by xw-g** [comment]
ATE test needs update too? @trathod1

**Comment by xw-g** [comment]
I"m doing presubmit effort via internal b/273579612

**Review by xw-g — APPROVED** [review]

**Pull request merged**